### PR TITLE
[Support/Stacks] Send many operations included in operation history

### PR DESCRIPTION
**Pull request by lawRathod**
### 📝 Description

Add support for send-many operations to appear in the operation history. 

### ❓ Context

- **JIRA or GitHub link**: [https://ledgerhq.atlassian.net/browse/LIVE-10786](https://ledgerhq.atlassian.net/browse/LIVE-10786)

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: libs/ledger-live-common/src/families/stacks/bridge/utils/misc.ts [hunk 1]
```diff
@@ -65,71 +66,103 @@ export const getAddress = (a: Account): Address =>
     : { address: a.freshAddress, derivationPath: a.freshAddressPath };
 
 export const mapTxToOps =
```
Comment (hedi-edelbloute) [inline libs/ledger-live-common/src/families/stacks/bridge/utils/misc.ts:68]: Could you please add a unit test for "send-many" (and maybe standard "token_transfer" ) case ? :pray: so we keep knowledge on how it's treated

**Review by hedi-edelbloute — COMMENTED** [review]
Lgtm overall !

**Review by hedi-edelbloute — DISMISSED** [review]

File: libs/ledger-live-common/src/families/stacks/bridge/utils/misc.ts (hunk 1 above)
Comment (lawRathod) [reply libs/ledger-live-common/src/families/stacks/bridge/utils/misc.ts:68]: Hey @hedi-edelbloute!! Added some unit tests and also the bridge tests snapshots now include send-many txns.

Comment (hedi-edelbloute) [reply libs/ledger-live-common/src/families/stacks/bridge/utils/misc.ts:68]: @lawRathod perfect thanks :ok_hand:

**Review by hedi-edelbloute — DISMISSED** [review]

**Review by hedi-edelbloute — APPROVED** [review]

**Pull request merged**